dashboard/test/ui/features/platform/header.feature

```diff
@@ -0,0 +1,133 @@
+@no_mobile
```

Comment:
git is getting confused about me moving this file and indenting all the lines to match the other files.

I indented the lines and only added the last scenario.